frontend/components/ButtonGroup.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button } from "antd";
+import { useSearchConfigStore } from "@/stores";
+
+const ButtonGroup: React.FC = () => {
+  const resetSearchConfig = useSearchConfigStore((state) => state.reset);
+
+  return (
+    <div className="flex flex-row justify-center my-4">
+      <Button className="mr-2" onClick={() => void resetSearchConfig()}>
```

Comment:
It looks weird.
how about `onClick={() => resetSearchConfig()}` or `onClick={resetSearchConfig}`?

Reply:
This is to avoid intended side effects: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void#non-leaking_arrow_functions.